test/math_test.py

```diff
@@ -1046,6 +1046,13 @@ def test_project_v2_raises_if_other_is_not_iterable(self):
 
         # act / assert
         self.assertRaises(TypeError, v.project, other)
+        
+    def test_clamp_mag_v2(self):
```

Comment:
I miss a test for the clamp_magnitude  the minimum. 
Sometimes it helps to think about the limits and write tests especially for those. So in this case I would expect:

1. a test to clamp to a minimum
2. a test to clamp to the maximum
3. a test where the vector is left as it is (no clamping)

And of course for each implemented method (v2.clamp_magnitude, v2.clamp_magnitude_ip, v3.clamp_magnitude, v3.clamp_magnitude_ip)